Dockerfile

```diff
@@ -3,6 +3,16 @@ FROM public.ecr.aws/docker/library/ruby:2.7.6
 # Throw errors if Gemfile has been modified since Gemfile.lock
 RUN bundle config --global frozen 1
 
+# Install cron
+RUN apt-get -y update && apt-get install -y cron
+# Create the log file to be able to store logs
+RUN touch /var/log/cron.log
+# Cron (at least in Debian) does not execute crontabs with more than 1 hardlink, see bug 647193.
+# As Docker uses overlays, it results with more than one link to the file, so you have to touch
+# it in your startup script, so the link is severed.
```

Comment:
🤔 I'm sure this would be super clear to someone with more cron experience than I have, but could you explain how the creation of this file here severs the link??